Algorithmia/algorithm.py

```diff
@@ -41,31 +41,23 @@ def set_options(self, timeout=300, stdout=False, output=OutputType.default, **qu
     def create(self, details={}, settings={}, version_info={}):
         detailsObj = Details(**details)
         settingsObj = SettingsMandatory(**settings)
-        createRequestVersionInfoObj = CreateRequestVersionInfo(**version_info)
+        createRequestVersionInfoObj = VersionInfo(**version_info)
         create_parameters = {"name": self.algoname, "details": detailsObj, "settings": settingsObj, "version_info": createRequestVersionInfoObj}
         create_request = CreateRequest(**create_parameters)
-        try:
```

Comment:
This try/catch was removed as the intended purpose of RaiseAlgoApiErrors was to handle algorithm request errors. Those have been removed as they swallow critical stack traces and other important pieces of information